src/components/codeblock.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import { Box } from '@blockstack/ui';
+import css from '@styled-system/css';
+import { Highlighter } from '@components/highlighter';
+
+export const CodeBlock = ({ code }: { code: string }) => {
+  const [editorCode] = useState(code?.toString().trim());
+
+  return (
+    <Box
+      // @ts-ignore
+      css={css({
+        '*': {
+          whiteSpace: 'pre',
+          fontFamily: 'Fira Code',
```

Comment:
Should fallback to Consolata, monospace etc, most won't have Fira Code installed.

Reply:
yeah, should add a fallback. We are loading the font, too, so folks will have it.